src/pymongo_migrate/mongo_migrate.py

```diff
@@ -123,7 +133,9 @@ def upgrade(self, migration_name: Optional[str] = None):
                 )
                 continue
             self.logger.info("Running upgrade migration %r", migration.name)
-            migration.upgrade(self.db)
+            with measure_time as elapsed:
+                migration.upgrade(self.db)
+                self.logger.info(f"Execution time of {migration.name}: {elapsed()}s")
```

Comment:
I used here an `f-string` to pass content to the logger, although the rest of the file usage `%r` style. Is it ok?

Reply:
know I think about it - no, please use %-format style

my reason for sticking with %-format style for logging is the way tools like sentry handle error log aggregation - as it integrates with logging on Python level it can easily aggregate stuff using the same format string. If you use f"" formatting each message looks like a different one to it and it cannot aggregate.

So yea, %-format is not as nice, but preventing duplicates in Sentry is much bigger concern to me.

Reply:
Fixed.